src/modules/organization/home/team/org-event-list-card/useOrgEventListCard.ts

```diff
@@ -0,0 +1,15 @@
+export default function useOrgEventListCard(
+  attendeeLimit: number,
+  currentAttendee: number,
+  eventStatus: string
+) {
+  const participantNumber = currentAttendee + "/" + attendeeLimit;

```

Comment:
use computed for all of these

Reply:
But they are static values which we know will never change (because we only get from API once only when the page is loaded), so why use computed
[](https://dev.to/pecus/when-to-avoid-vuejs-computed-properties-for-greater-performance-5anj)